example_code/basic_examples/fallback_receive/src/lib.rs

```diff
@@ -0,0 +1,245 @@
+#![cfg_attr(not(feature = "export-abi"), no_std)]
+extern crate alloc;
+
+use stylus_sdk::alloy_sol_types::sol;
+use stylus_sdk::{
+    alloy_primitives::{U256, Address, FixedBytes},
+    ArbResult,
+    prelude::*,
+};
+use alloc::vec;
+use crate::alloc::string::ToString;
+use crate::vec::Vec;
+// Define persistent storage
+sol_storage! {
+    #[entrypoint]
+    pub struct PaymentTracker {
+        uint256 total_received;
+        uint256 fallback_calls;
+        uint256 receive_calls;
```

Comment:
```suggestion
        uint256 totalReceived;
        uint256 fallbackCalls;
        uint256 receiveCalls;
```

I believe this definition should use camel case because it is in Solidity.